updatecli/updatecli.d/terraform-providers.yaml

```
@@ -27,12 +27,11 @@ targets:
       file: ".terraform.lock.hcl"
     scmID: default
 
-pullrequests:
+actions:
   default:
-    kind: github
-    scmID: default
-    targets:
-      - upgradeActualLockFile
+    kind: github/pullrequest
+    scmid: default
```

```suggestion
    scmid: default
    title: "Bump Terraform plugins - not modules"
```
Also, `title` at line 1 have to be renamed `name`

---

To be tested but the renaming might be the only needed change.

---

I would says so as I took example on this one : https://github.com/jenkins-infra/azure/blob/main/updatecli/updatecli.d/terraform-providers.yaml